server/adaptors/integrations/__data__/repository/aws_elb/schemas/aws_elb-1.0.0.mapping.json

```diff
@@ -0,0 +1,98 @@
+{
+  "template": {
+    "mappings": {
+      "_meta": {
+        "version": "1.0.0",
+        "catalog": "observability",
+        "type": "logs",
+        "component": "aws_elb"
+      },
+      "properties": {
+        "aws": {
+          "type": "object",
+          "properties": {
+            "elb": {
+              "properties": {
+                "backend.ip": {
+                  "type": "ip"
+                },
+                "backend.port": {
+                  "type": "integer"
+                },
+                "backend.processing_time": {
+                  "type": "half_float"
+                },
+                "backend.status_code": {
+                  "type": "short"
+                },
+                "client.ip": {
+                  "type": "ip"
+                },
+                "client.port": {
+                  "type": "integer"
+                },
+                "connection_time": {
+                  "type": "integer"
+                },
+                "destination.ip": {
+                  "type": "ip"
+                },
+                "destination.port": {
+                  "type": "integer"
+                },
+                "elb_status_code": {
+                  "type": "short"
+                },
+                "http.port": {
+                  "type": "integer"
+                },
+                "http.version": {
+                  "type": "keyword"
+                },
+                "matched_rule_priority": {
+                  "type": "integer"
+                },
+                "received_bytes": {
+                  "type": "integer"
+                },
+                "request_creation_time": {
+                  "type": "date"
+                },
+                "request_processing_time": {
+                  "type": "half_float"
+                },
+                "response_processing_time": {
+                  "type": "half_float"
+                },
+                "sent_bytes": {
+                  "type": "integer"
+                },
+                "ssl_protocol": {
+                  "type": "keyword"
+                },
+                "ssl_cipher": {
+                  "type": "keyword"
+                },
+                "target_ip": {
+                  "type": "ip"
+                },
+                "target_port": {
+                  "type": "integer"
+                },
+                "target_processing_time": {
+                  "type": "half_float"
```

Comment:
where is this `half_float` coming from ?
why not using `long` ?

Reply:
It's a float in the data.

Reply:
Should I change it to float instead?

Reply:
half_float is fine